app/models.py

```diff
@@ -1467,6 +1467,7 @@ def serialize(self):
         data.update({
             'id': self.id,
             'briefId': self.brief_id,
+            'briefTitle': self.brief.data.get('title', ''),
```

Comment:
Only just seen your PR comment (think we posted our things at very similar times). Good examples so it looks like we are doing this a fair amount so I guess this is acceptable (although I might raise it as a discussion point in the next tech debt meeting).